dapp/src/components/ModalRoot/withBaseModal.tsx

```diff
@@ -1,13 +1,21 @@
 import React, { ComponentType, useCallback } from "react"
-import { Modal, ModalContent, ModalOverlay, ModalProps } from "@chakra-ui/react"
+import {
+  BoxProps,
+  Modal,
+  ModalContent,
+  ModalOverlay,
+  ModalProps,
+} from "@chakra-ui/react"
 import { BaseModalProps } from "#/types"
 import { useAppNavigate, useSidebar } from "#/hooks"
 
 const MODAL_BASE_SIZE = "lg"
 
 function withBaseModal<T extends BaseModalProps>(
   WrappedModalContent: ComponentType<T>,
-  modalProps?: Omit<ModalProps, "isOpen" | "onClose" | "children">,
+  modalProps?: Omit<ModalProps, "isOpen" | "onClose" | "children"> & {
+    border?: BoxProps["border"]
```

Comment:
As I mentioned in another comment, we can simply disable the border for all modal windows.